go.mod

```diff
@@ -3,23 +3,20 @@ module github.com/fluxcd/helm-operator
 go 1.13
 
 require (
-	github.com/bugsnag/bugsnag-go v1.5.3 // indirect
+	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/fluxcd/flux v1.15.0
```

Comment:
Shouldn't this also be updated?

Reply:
It's an indirect, I can pin it but I see no point in doing so.